modules/cnf-installing-sriov-fec-operator.adoc

```diff
@@ -0,0 +1,170 @@
+----
+
+.. Create the `OperatorGroup` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-operatorgroup.yaml
+----
+
+.. Run the following command to get the `channel` value required for the next step.
++
+[source,terminal]
+----
+$ oc get packagemanifest sriov-operator -n openshift-marketplace -o jsonpath='{.status.defaultChannel}'
+----
++
+.Example output
+[source,terminal]
+----
+stable
+----
+
+.. Create the following `Subscription` CR and save the YAML in the `sriov-sub.yaml` file:
++
+.Example Subscription
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+    name: sriov-fec-subscription
+    namespace: vran-acceleration-operators
+spec:
+    channel: "<channel>" <1>
+    name: sriov-fec
+    source: certified-operators <2>
+    sourceNamespace: openshift-marketplace
+----
+<1> Specify the value for channel from the value obtained in the previous step for the `.status.defaultChannel` parameter.
+<2> You must specify the `certified-operators` value.
+
+.. Create the `Subscription` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-sub.yaml
+----
+
+.. Verify that the Operator is installed:
++
+[source,terminal]
+----
+$ oc get csv -n vran-acceleration-operators -o custom-columns=Name:.metadata.name,Phase:.status.phase
+----
++
+.Example output
++
+[source,terminal]
+----
+Name                                        Phase
+sriov-fec.v1.1.0                            Succeeded
+----
+
+[id="installing-openness-sriov-fec-operator-using-web-console_{context}"]
+== Installing the OpenNESS SR-IOV Operator for Wireless FEC Accelerators by using the web console
+
+As a cluster administrator, you can install the OpenNESS SR-IOV Operator for Wireless FEC Accelerators by using the web console.
+
+[NOTE]
+====
+You must create the `Namespace` and `OperatorGroup` CR as mentioned in the previous section.
+====
+
+.Procedure
+
+. Install the OpenNESS SR-IOV Operator for Wireless FEC Accelerators by using the {product-title} web console:
+
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+.. Choose *OpenNESS SR-IOV Operator for Wireless FEC Accelerators* from the list of available Operators, and then click *Install*.
+
+.. On the *Install Operator* page, select *All namespaces on the cluster*. Then, click *Install*.
+
+. Optional: Verify that the SRIOV-FEC Operator is installed successfully:
+
+.. Switch to the *Operators* -> *Installed Operators* page.
+
+.. Ensure that *OpenNESS SR-IOV Operator for Wireless FEC Accelerators* is listed in the *vran-acceleration-operators* project with a *Status* of *InstallSucceeded*.
++
+[NOTE]
+====
+During installation an Operator might display a *Failed* status. If the installation later succeeds with an *InstallSucceeded* message, you can ignore the *Failed* message.
+====
++
+If the Operator does not appear as installed, to troubleshoot further:
```

Comment:
Maybe "If the console does not indicate that the Operator is installed, perform the following troubleshooting steps:"